src/AppBundle/GraphQL/Resolver/CharacterResolver.php

```diff
@@ -18,13 +20,42 @@ class CharacterResolver extends BaseManagerService implements ContainerAwareInte
     public function resolve(Argument $args = null)
     {
         $characterType = null;
-        
+
         if (isset($args["characterId"])) {
             $characterType = CharacterType::fromId($this->game, (int)$args["characterId"]);
         } elseif (isset($args["characterName"])) {
             $characterType = CharacterType::fromName($this->game, $args["characterName"]);
         }
-        
+
         return $characterType;
     }
+
+    /**
+     * Returns a graphql character type from a cursor.
+     * @param type $args Arguments array, as given by CharacterConnection->getEdges()
+     * @return CharacterType
+     */
+    public function getCharacterFromCursor($args = null): CharacterType
+    {
+        $cursor = $args["cursor"];
+        $user = $args["__data"];
+
+        $offset = CharacterConnection::decodeCursor($cursor);
```

Comment:
Is there error handling needed? If so, let's at least file an issue or put a TODO so we remember to come back.